README.md

```diff
@@ -0,0 +1,39 @@
+# Hapi-OpenAM
+
+### Integrating OpenAM with Hapi authentication plugins
+
+This provides the validateFunc for the following hapi plugins:
+```
+hapi-auth-basic
+hapi-auth-bearer-token
+```
+
+### Example usage
+```
+server = new Hapi.Server();
+server.connection({ port: 5050});
+server.register([
+  {register: require('inject-then')},
```

Comment:
Awesome job creating a README. Registering this plugin might confuse some readers as 1. we don't use it in the rest of the example and 2. it's not essential for the code to work.